frontend/src/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
This really shouldn't be using a string. Definitely need to refactor when setting status from the status toggle.